Training25/Program.cs

```diff
@@ -16,8 +16,9 @@ static void Main () {
          WriteLine ("Enter two positive integers: ");
          if (int.TryParse (ReadLine (), out int num1) && num1 > 0 &&
             int.TryParse (ReadLine (), out int num2) && num2 > 0) {
-            WriteLine ($"GCD of {num1} and {num2} is {GCD (num1, num2)}");
-            WriteLine ($"LCM of {num1} and {num2} is {LCM (num1, num2)}");
+            int gcd = GCD (num1, num2);
+            WriteLine ($"GCD of {num1} and {num2} is {gcd}");
+            WriteLine ($"LCM of {num1} and {num2} is {LCM (gcd, num1, num2)}");

```

Comment:
```suggestion
            WriteLine ($"LCM of {num1} and {num2} is {(num1 * num2) / gcd}");
```
No need for a separate LCM function